src/components/data/FileBrowser.js

```diff
@@ -472,7 +472,7 @@ const BucketBrowser = (({
       }),
 
       creatingNewFolder && h(NameModal, {
-        thing: 'New folder',
+        thing: 'Folder',
```

Comment:
Tangentially related. NameModal includes "New" in the modal title, so currently this modal is titled "Create a New New Folder".

https://github.com/DataBiosphere/terra-ui/blob/2244d930ec64ed3b895dcb470d2bec144dec8b6a/src/components/NameModal.js#L27